src/TLuaInterpreter.cpp

```diff
@@ -1912,16 +1804,9 @@ int TLuaInterpreter::setWindowWrap(lua_State* L)
     int s = 1;
     QString windowName;
     if (lua_gettop(L) > 1) {
-        windowName = WINDOW_NAME(L, 1);
-        ++s;
+        windowName = WINDOW_NAME(L, s++);
     }
-
-    if (!lua_isnumber(L, s)) {
-        lua_pushfstring(L, "setWindowWrap: bad argument #%d type (wrapAt as number expected, got %s!)", s, luaL_typename(L, s));
-        return lua_error(L);
-    }
-    int luaFrom = lua_tointeger(L, 2);
-
+    int luaFrom = getVerifiedInt(L, __func__, s, "wrapAt");
```

Comment:
Indeed it would be, also these are the internal variable names, so to me seem even less important than the names shown to the players which are to be aligned in #4638